graphql/database/users/__tests__/createUser.test.js

```diff
@@ -1116,4 +1126,52 @@ describe('createUser when user already exists (should be idempotent)', () => {
 
     expect(returnedUser.searchEngine).toEqual('SearchForACause')
   })
+
```

Comment:
Seems like the v1 test is no longer testing v1 because it's using the same getUserFeature mock set up in `beforeEach` that has the v2 experiment enabled? If that's right, explicitly set the experiment mock there to keep the test to v1.

Reply:
I think we still need to return something since `v2` exists, but that something should be correct now in the `beforeEach`.

Reply:
Yup, looks good 👍